cpp/src/io/json/json_column.cu

```diff
@@ -38,6 +38,7 @@
 #include <thrust/for_each.h>
 #include <thrust/functional.h>
 #include <thrust/gather.h>
+#include <thrust/iterator/constant_iterator.h>
```

Comment:
I think this header isn't needed here anymore but instead should be included in `json_tree.cu` now.